### PR TITLE
Backport v9 shell CLI command

### PR DESCRIPTION
**Pull request by dreispt**
Find more about it at: https://github.com/odoo/odoo/pull/4145

### CODE REVIEW

**Comment by antonylesuisse** [comment]
Move sell.py to cli/ so that it will compatible with v9, if we agree on this convention.

**Comment by dreispt** [comment]
Did a rebase from master and included the changes in PR odoo/odoo#4531 to make Travis happy.

To test this, you first need to comment the code in `server-tools/server_environment/__init__.py`, at least until #89 / #93 is solved.
Then just type: `./odoo.py shell -d <dbname>`.

**Comment by dreispt** [comment]
Ping!

**Comment by pedrobaeza** [comment]
Can you please add a README.rst explaning the use?

**Comment by dreispt** [comment]
Added a README and rebased.

File: shell/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Makes available in Odoo 8 the `shell` server command available for Odoo 9.
```
Comment (pedrobaeza) [inline shell/README.md:1]: Please put a title

File: shell/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+To use, you should have this module in the addons path, 
+and then in a terminal window run `./odoo.py shell -d <dbname>`.
+
+This will initialize a server instance and then jump into a Pyhton
+interactive shell, with full access to the Odoo API.
```
Comment (pedrobaeza) [inline shell/README.md:9]: Can you use OCA template?

File: shell/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Makes available in Odoo 8 the `shell` server command available for Odoo 9.
+
+To use, you should have this module in the addons path,
```
Comment (pedrobaeza) [inline shell/README.md]: s/use/use it/

File: shell/README.md (hunk 2 above)
Comment (pedrobaeza) [inline shell/README.md:9]: An example of what you can put in this interactive sheel would be great.

**Comment by pedrobaeza** [comment]
Sorry for being so insistent..., but I think this will improve a lot the knowledge about the module and avoid future questions.

**Comment by dreispt** [comment]
Edited the README and rebased.
AFAIK there is no mandatory template for module descriptions. (To be honest, I'm not aware if this template at all :-/ ).

Note that this is a backport from v9, not an absolutely new feature.

**Comment by pedrobaeza** [comment]
Thanks :+1:

**Comment by dreispt** [comment]
Is this OK to merge?

**Comment by pedrobaeza** [comment]
Another approval required and a little change that I put (title on the README)

**Comment by bwrsandman** [comment]
:+1:

**Comment by pedrobaeza** [comment]
I merge and added the title in README.md

**Pull request merged**